koku/masu/external/downloader/ocp/ocp_report_downloader.py

```diff
@@ -112,6 +120,41 @@ def get_report_for(self, date_time):
 
         return reports
 
+    def detect_type(self, data_frame):
```

Comment:
Would be cool to consolidate this report detection to an OCP utility file one day:
https://github.com/project-koku/koku/blob/368f317a025b905210af92c3cddfeeca5bf143dc/koku/masu/processor/ocp/ocp_report_processor.py#L147

Reply:
Done. Let me know if it doesn't work for you.